.github/workflows/pre-commit.yml

```diff
@@ -24,4 +24,4 @@ concurrency:
 jobs:
   call-workflow-pre-commmit:
     name: Pre-Commit
-    uses: MiraGeoscience/CI-tools/.github/workflows/reusable-pre_commit.yml@main
+    uses: MiraGeoscience/CI-tools/.github/workflows/reusable-pre_commit.yml@DEVOPS-565-fixup
```

Comment:
I will change back to main before merging

Reply:
Wouldn't you rather reverse Andrew's merger?